src/fragments/lib/graphqlapi/js/setup/connect.mdx

```diff
@@ -1070,33 +1175,13 @@ const promise = makeAPICall();
 API.cancel(promise, "my error message");
 ```
 
-### Recap
-You now have an understanding of how to read your data through get and list queries. 
-
-[comment]: # (ADD TO RECAP a tie-in to real-world use and application of the topic discussed OR reiterate the benefit.)
-
+**Recap:** You now have an understanding of how to read your data through `Get` and `List` queries. 
 
 ## Subscribe to real-time events
 
-In this job, you will subscribe to real-time events. This includes understanding the benefits of enabling real-time data integrations and how to set up and filter these subscriptions. We will also cover how to unsubscribe from subscriptions when needed.
-
-> **Before you begin, you will need:**
-> - Your application connected to the API
-> - Populated data to modify
+You can subscribe to real-time events to help manage your data. In the section below we outline the benefits of enabling real-time data integrations and how to set up and filter these subscriptions. We will also cover how to unsubscribe from subscriptions when needed. Before you begin you will need your application connected to the API and populated data to modify.
```

Comment:
In our workshop we discussed adding some context around the befits of enabling real-time data integrations. There isn't anything new here - is there more we wanted to add here or does this cover it?